.gitlab-ci.yml

```diff
@@ -372,7 +374,6 @@ build-linux-substrate:             &build-binary
   stage:                           build
   <<:                              *collect-artifacts
   <<:                              *docker-env
-  <<:                              *build-only
```

Comment:
OK, I see now that you've enabled the build job for every commit. But also it enables `build-linux-subkey` job for every commit. See https://gitlab.parity.io/parity/substrate/pipelines/94278